dotcom-rendering/fixtures/manual/consentStateCanTarget.ts

```diff
@@ -0,0 +1,310 @@
+import type { ConsentState } from '@guardian/consent-management-platform/dist/types';
+
+export const consentStateCanTarget: ConsentState = {
```

Comment:
I think we could replace this with much smaller mocked consentstate like this:

https://github.com/guardian/commercial/blob/ba0ffd3cdefdc700bf1e204f5954a2a16a4044d3/src/core/targeting/build-page-targeting.spec.ts#L91-L99

And then remove this file and just inline it for the tests it used in. 

Non-blocking though and I can pick this up as a follow up if you want to get this in.